subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -830,8 +830,27 @@ int ull_conn_llcp(struct ll_conn *conn, u32_t ticks_at_expire, u16_t lazy)
 		}
 	}
 
-	/* check if procedure is requested */
+	/* Check if procedures with instant or encryption setup is requested or
+	 * active.
+	 */
 	if (((conn->llcp_req - conn->llcp_ack) & 0x03) == 0x02) {
+		/* Process parallel procedures that are active */
+		if (0) {
```

Comment:
why the 'if (0)'?

Reply:
Ah! A style I follow when having many conditionally compiled if-then block. But, here I should not be using it. If you have a strong feeling I should change it, please request change.

Reply:
No strong feeling, let's get this in so we can proceed